History.md

```diff
@@ -1,6 +1,13 @@
 Dalli Changelog
 =====================
 
+2.7.10
+==========
+- Revert frozen string change (schneems)
+- Advertise supports_cached_versioning? in DalliStore (schneems)
+- Better detection of fork support, to allow specs to run under Truffle Ruby
+- Update logging for over max size to log as error
```

Comment:
@petergoldstein 
Would you mind if you could give credit for @aeroastro and @deepj here?

It would be like this.

```markdown
- Better detection of fork support, to allow specs to run under Truffle Ruby (deepj, #712)
- Update logging for over max size to log as error (Takumasa Ochi, #715)
```

The original pull requests are here.
#712 #715 


Although you have replaced each Pull Request with your commit, IMHO, it would be really grateful if you could give credit to original contributors. I believe giving legitimate credit would foster future contributions and contribute to the quality of Dalli as a gem.

Reply:
Sure.  An oversight on my part.

I added the contributor reference in the manner Dalli has done historically - just by referencing the Github username.

Reply:
Thank you ❤️